src/config.ts

```diff
@@ -18,11 +18,18 @@ export async function loadBumpConfig(
   overrides?: Partial<VersionBumpOptions>,
   cwd = process.cwd(),
 ) {
+  const { config: bumppConfig } = await loadConfig<VersionBumpOptions>({
+    name: 'bumpp',
```

Comment:
After a second thought, I think it's better to only read from `bump` to avoid confusion,  as we already fix the docs for quite a while now.

Reply:
Thanks, LGTM